### PR TITLE
copy edits

### PR DESCRIPTION
**Pull request by rosemarymarano**
Reviewed and edited samples README files.

### CODE REVIEW

File: kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md [hunk 1]
```diff
@@ -67,20 +67,20 @@ The following parameters can be provided in the inputs file.
...
-| `createDomainFilesDir` | Directory to get all the create domain scripts and supporting files, including the script that is specified by the createDomainScriptName property. By default, this directory is set to the relative path `wlst`, and the create script will use the built-in WLST offline scripts in `wlst` directory to create the WebLogic domain. It can also be set to the relative path `wdt`, and then the built-in WDT scripts will be used instead. An absolute path is also supported to point to an arbitrary directory in the file system. | wlst |
+| `createDomainFilesDir` | Directory to get all the create domain scripts and supporting files, including the script that is specified by the `createDomainScriptName` property. By default, this directory is set to the relative path `wlst`, and the create script will use the built-in WLST offline scripts in the`wlst` directory to create the WebLogic domain. It can also be set to the relative path `wdt`, and then the built-in WDT scripts will be used instead. An absolute path is also supported to point to an arbitrary directory in the file system. | `wlst` |
```
Comment (doxiao) [inline kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md]: in the`wlst` directory -> in the `wlst` directory.

Comment (rosemarymarano) [reply kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/README.md]: Got it, thanks.

**Comment by doxiao** [comment]
LGTM.

**Comment by rosemarymarano** [comment]
Thanks.

-- 
http://www.oracle.com/
Rosemary Marano | Principal Technical Writer
Phone: HYPERLINK "tel:+16038973107"+16038973107 
Oracle User Assistance
One Oracle Drive | Nashua, New Hampshire 03062 

http://www.oracle.com/commitment

Oracle is committed to developing practices and products that help protect the environment

**Pull request merged**